src/main/java/com/epam/izh/rd/online/repository/SimpleAuthorRepository.java

```diff
@@ -0,0 +1,69 @@
+package com.epam.izh.rd.online.repository;
+
+import com.epam.izh.rd.online.entity.Author;
+
+import java.util.Arrays;
+
+public class SimpleAuthorRepository implements AuthorRepository{
+    private Author[] authors = new Author[0];
+
+    @Override
+    public boolean save(Author author) {
+        if (findByFullName(author.getName(), author.getLastName()) != null) {
+            return false;
+        }
+        resize();
+        authors[authors.length - 1] = author;
+        return true;
+    }
+
+    @Override
+    public Author findByFullName(String name, String lastname) {
+        for (Author author: authors) {
+            if (author.getName().equals(name) && author.getLastName().equals(lastname)) {
+                return author;
+            }
+        }
+        return null;
+    }
+
+    @Override
+    public boolean remove(Author author) {
+        for (int i = 0; i < authors.length; i++) {
+            if (authors[i].equals(author)) {
+                authors[i] = null;
+                resize();
+                return true;
+            }
+        }
+        return false;
+    }
+
+    //можно ли завязываться на null? Например если что-то не нашли, то вернуть null?
```

Comment:
yes, it valid but outdated approach. Many basic java classes use it (e.g. Map.get(String key)). But for now the best approach is to return Optional.empty() instead of null for methods where 'nothing' could be returned.